react/src/components/AssetDetail/AssetButton.tsx

```diff
@@ -28,11 +28,16 @@ const AssetButton: React.FC<AssetButtonProps> = ({
   const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
   const featureType = getFeatureType(selectedFeature);
   const projectId = selectedFeature.project_id;
+  const [isSuccess, setIsSuccess] = useState<boolean>(false);
   const notification = useNotification();
   const featureId = selectedFeature.id;
   const { mutate: importFeatureAsset, isPending: isImporting } =
     useImportFeatureAsset(projectId, featureId);
 
+  useEffect(() => {
+    setIsSuccess(false);
+  }, [selectedFeature.id]);
```

Comment:
```suggestion
  const notification = useNotification();
  const featureId = selectedFeature.id;
  const {
    mutate: importFeatureAsset,
    isPending: isImporting,
    isSuccess,
    reset,
  } = useImportFeatureAsset(projectId, featureId);

  useEffect(() => {
    reset();
  }, [selectedFeature.id, reset]);
```
Good call on the `useEffect` for feature change. I think we can use the mutation's `reset` method to reset the state whenever a new feature is selected